ipc/provider/src/manager/evm/manager.rs

```diff
@@ -102,6 +102,34 @@ impl TopDownCheckpointQuery for EthSubnetManager {
             .to_fixed_bytes()
             .to_vec())
     }
+
+    async fn get_validator_changeset(
+        &self,
+        subnet_id: &SubnetID,
+        start: ChainEpoch,
+        end: ChainEpoch,
+    ) -> Result<Vec<StakingChangeRequest>> {
+        let address = contract_address_from_subnet(subnet_id)?;
+        log::info!("querying validator changes in evm subnet contract: {address:}");
+
+        let contract = subnet_actor_manager_facet::SubnetActorManagerFacet::new(
+            address,
+            Arc::new(self.ipc_contract_info.provider.clone()),
+        );
+
+        let ev = contract
```

Comment:
So here you are getting all events that trigger __any__ change in membership, right?